### DISCUSSION TITLE
how to avoid prefixing Hubot's name in every message in a one on one conversation ?

### DISCUSSION THREAD

**Discussion by kavya05** [hubotio/hubot#1600] [Q&A]
hey i have developed a bot for my company and the participants in the chat are just "bot" and the "employee" so its very annoying for the employee to mention bots name at the beginning of each sentence , is there a way where they can converse with the bot in regular manner without having to mention bots name on every conversation?
its very urgent for me i'm reaching my deadlines so please help me out in this one
example:
employee: hubot show my storage space resources
instead he should be able to say
employee: show my storage space resources.

**Comment by xurizaemon** [comment]
From my recollection this is (or can be) an adapter feature, and at least one of the adapters for IRC, Rocket.chat or Mattermost implements it.
